tsconfig.json

```diff
@@ -1,16 +1,16 @@
 {
     "compilerOptions": {
       "lib": [
-          "ES2017",
+          "ES2021",
           "webworker"
       ],
       "sourceMap": true,
       "rootDir": "src",
       "outDir": "dist",
-      "module": "ESNext",
+      "module": "NodeNext",
```

Comment:
why `NodeNext` here and line 13

AFAIK `node16` is enough and not experimental

https://github.com/containers/podman-desktop/blob/5c9313c29e2a1144424ed406010b3c46b6481dc7/packages/main/tsconfig.json#L4